README.md

```diff
@@ -99,13 +99,13 @@ Then, all that remains is to call `onRemoteClipboardChanged` every time the clip
 In the unfortunate case of anticipated local changes to the clipboard done in the background, this can be improved in two ways:
 
 * Regular polling of the token and invoking a similar handler to the `focus` handler in the snippet above: this is generally not the best solution, but this API should be lightweight enough that it doesn’t create much overhead.
-* Integrating this with `clipboardchange` event in addition (or instead) or the `focus` event: this depends on whether `clipboardchange` event becomes a part of the web standard.
+* Integrating this with `clipboardchange` event in addition (or instead) or the `focus` event: this depends on whether `clipboardchange` event becomes a part of the web standard and requires to somehow ignore the listener call between `write` and `contentsID`.
```

Comment:
I'd make the second concern a second sentence, since it's not related to the uncertainty around `clipboardchange`:

```suggestion
* Integrating this with `clipboardchange` event in addition (or instead) or the `focus` event: this depends on whether `clipboardchange` event becomes a part of the web standard.
  This API's design will need to be integrated with the
  `clipboardchange` design to ensure it isn't delivered between
  writing to the clipboard and updating the last-known token value.
```

Reply:
Not sure how the design could be integrated, as there's no clear way for the browser to correlate `clipboarchange` that originates from the system to `write` - other than maybe adding an option to ignore first event after performing `write`? Amended wording a bit, what do You think?